config.go

```diff
@@ -59,6 +59,9 @@ type config struct {
 	CPUProfile string `long:"cpuprofile" description:"Write CPU profile to the specified file"`
 	MemProfile string `long:"memprofile" description:"Write mem profile to the specified file"`
 
+	// Monitor options
+	Monitor string `long:"monitor" description:"Enable a JSON API status monitor on given port -- NOTE port must be between 1024 and 65536"`
```

Comment:
You don't need the note about port numbers since that is unrelated to this and is really just a general UNIX port thing (requiring additional privileges for ports <1024).

Reply:
Do you think it's good to add a `short` config for this?

Reply:
I'd prefer if it were a string ```[]string```.  Maybe call it APIListeners and copy & paste the code from:

https://github.com/decred/dcrd/blob/539fb844cca27456ce02b5185c97e79e63183798/config.go#L748-L758

and replace RPCListeners with APIListeners.

We should give a bit more control in case someone is running on a box with multiple interfaces since there isn't any authentication.

Reply:
If the most obvious letter is still available, I'm always in favor of a short option.